### DISCUSSION TITLE
Indicate target registry on publish

### DISCUSSION THREAD

**Discussion by harrisonepperson** [npm/feedback#243] [CLI] [answered]
**Issue:**
NPM extends multiple methods to set a custom registry, and when publishing we are provided a list of files bundled for publication; however, we receive no indication of what registry this module is published to.  This can get confusing when publishing to multiple registries within a pipeline.

**Suggestion:**
When outputting the publish review information, consider adding which registry the package was directed to.  This should also apply when using the --dry-run flag

**Comment by MylesBorins** [comment, marked as answer]
@harrisonepperson thanks for the suggestion!

The next step in getting something like this in the npm CLI would be opening an RFC issue at https://github.com/npm/rfcs to suggest the addition to the team.
